### ISSUE TITLE
hyperlinks to fn do not function in pdf

### ISSUE THREAD

**Issue by leighw** [dita-ot/dita-ot#2359]
Tested with OT 2.2.4, dita2pdf transform. Given this syntax in a topic:

```xml
<p>This is a paragraph with a footnote in it. <fn id="footnote-1">Here is a footnote.</fn>>/p>
<p>And here is a paragraph with a cross-reference to that footnote. <xref
 href="#topica/footnote-1" format="dita" type="fn"/></p>
```

The expected result is a functioning hyperlink to the footnote. No link is created. It's rendered in topic.fo as:
 
```xml
<fo:block space-after="0.6em" space-before="0.6em" text-indent="0em">Here is a
paragraph that cross-references a footnote.<fo:inline baseline-shift="super"
font-size="75%" keep-with-previous.within-line="always">1</fo:inline></fo:block>
```
In other words, plain superscripted text. The same syntax results in a functioning hyperlink in the dita2html transform.

**Comment by robander** [comment]
I can reproduce this - with my simple topic, I get the correct footnote (superscript "1") but it is not a link, and presumably should be. Marked as a bug though this could arguably be called a feature, as I would guess it has never worked, and was apparently not part of the original PDF design.

```xml
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE task PUBLIC "-//OASIS//DTD DITA Task//EN"
 "../../dtd/technicalContent/dtd/task.dtd">
<task id="changeoil" xml:lang="en-us">
<title>Changing the oil in your car</title>
<shortdesc>This isn't really about changing the oil anymore.</shortdesc>
<taskbody>
<context><p>Changing the oil regularly will help keep the engine in good condition. <fn id="testfn">this is a
test footnote</fn></p>
<p>Here is a link to the footnote: <xref href="#changeoil/testfn" format="dita" type="fn"/></p></context>
</taskbody>
</task>
```

**Comment by jelovirt** [comment]
Implemented in #2364
